gradle/versions.gradle

```diff
@@ -3,7 +3,7 @@ ext {
     android_sdk_versions_plugin = '1.0.0'
 
     dokka_version = '0.9.17'
-    kotlin_version = '1.3.61'
+    kotlin_version = '1.3.72'
 
     min_sdk_version = 23
     target_sdk_version = 28
```

Comment:
nit: 29
build_tools_version = 29.0.2

Reply:
Fixed.

Reply:
Well I haven't yet carefully reviewed implications on target sdk 29 https://developer.android.com/about/versions/10/behavior-changes-10

Reply:
I've merged it to dev because everything seemed to work at first glance.
I'll take a closer look tomorrow.